services/movie.py

```diff
@@ -28,13 +33,17 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
 ) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
-
-    return movie
+    try:
+        with transaction.atomic():
+            movie = Movie.objects.create(
+                title=movie_title,
+                description=movie_description,
+            )
+            if genres_ids:
+                movie.genres.set(genres_ids)
+            if actors_ids:
+                movie.actors.set(actors_ids)
+
+        return movie
+    except Exception as e:
+        raise ValueError(f"Error creating movie: {str(e)}")
```

Comment:
The exception handling here is very generic. Catching a general `Exception` can make it difficult to diagnose specific issues. Consider catching more specific exceptions related to database operations if possible.